src/HttpTunnel/HttpTunnel.csproj

```diff
@@ -11,15 +11,15 @@
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageReadmeFile>readme.md</PackageReadmeFile>
-    <Version>1.0.1</Version>
+    <Version>1.0.2</Version>
```

Comment:
Please change to 2.0.0 after we upgrade from .Net 5 to .Net 7. This is a major change and the previous setup may not work.